lib/send_sms.rb

```diff
@@ -0,0 +1,16 @@
+require 'twilio-ruby'
+
+time = Time.new + 3600 
+
+account_sid = "AC7c0dd1eab28593f5ec87bb03c409e40b"
+auth_token = "0b767001604b77a4e1c3e6585c909586"
+client = Twilio::REST::Client.new(account_sid, auth_token)
+
+from = '+447862152030' # Your Twilio number
+to = '+447725810503' # Your mobile phone number
+
+client.messages.create(
```

Comment:
might be more readable to have this section as a new method - keeps your code as single responsibility